flink-filesystems/flink-s3-fs-hadoop/pom.xml

```diff
@@ -182,6 +182,21 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<!-- for HadoopS3FileSystemITCase.testRecursiveUploadForYarn -->
```

Comment:
Would be great if we can avoid adding these dependencies.
This couples projects that were really meant to be independent, even if just in test scope.

If this is about testing recursive upload, can this be written properly as a test case in this project?
Or can the Yarn upload test be completely in the yarn test project, adding a dependency on this s3 project?

Reply:
Yes this is about the recursive upload which needs to be tested once with hdfs and once more with s3.

Sure we could flip the dependency and let the tests in the `yarn` sub-project depend on `flink-s3-fs-hadoop` (and I don't mind which depends on which, actually) but wouldn't this be just the same but in reverse?